garageScript/c0d3-app
# Highlight challenge codeblock and inline-code

### Description

The code in our challenges whether a code block or an inline code is hard to read because it lacks syntax highlighting and no background to distinguish it from other text. 

## Changes

- Update `ChallengeMaterial` component to override `pre` and `code` elements to our markdown components
- Update snapshots

## Screenshots

### Before

![image](https://user-images.githubusercontent.com/35906419/226459403-ed0ae4ee-5ee1-43d7-a5d7-362da0cc9edd.png)


### After

![image](https://user-images.githubusercontent.com/35906419/226459319-d1d3344e-f293-4fdd-b661-86afcd624450.png)

### Testing

1. Go to any curriculum challenges with a codeblock or inline-code.
2. Confirm the syntax is highlighted

### Related issues

- Closes #2819
